src/Modules/OrchardCore.Commerce/Assets/Scripts/stripe-card-form.js

```diff
@@ -0,0 +1,142 @@
+// Adding credit card element with Stripe API.
```

Comment:
Why are these files ignored by eslint?

> 0:0  warning  File ignored because of a matching ignore pattern. Use "--no-ignore" to override

If you force them not to be ignored, it shows these warnings, please fix them:
![image](https://user-images.githubusercontent.com/4007293/181949075-00b2310d-eb5b-4286-ba23-3e30e8335d7f.png)

Reply:
I turned it off on purpose. Now I fixed everything I can, but there are some unfixable things. I made comments about them in code and disabled eslint for them but that's not enough: in local it works but it fails here. So I turned off eslint for them again.

Reply:
It doesn't work because of this:
![image](https://user-images.githubusercontent.com/4007293/182022783-206f344f-eaad-45ef-8d94-0c8f7304982c.png)
Try adding eslint to the devDependencies, that should do the trick.

@0liver is this normal?

Reply:
It was already there by default: 
![image](https://user-images.githubusercontent.com/80963259/182028000-5aa29371-dd1e-4a6d-806a-d5fbf750517e.png)

Reply:
[brah, I added it](https://github.com/OrchardCMS/OrchardCore.Commerce/pull/127/commits/0bf22c548fd68f397acc3ab30bf6c6de9c7088a4)

Reply:
Didn't saw that, I was surprised that it worked after adding back what you deleted to [pr_ci.yml](https://github.com/OrchardCMS/OrchardCore.Commerce/pull/127/files/d3aa19e2d3ba25aa6187aa4834ddf3a6a50fc2f5#diff-1153a5676594ad20522a7686e7e94a258891fe87611bafcd10ec0f3943c219b8). Thanks!